src/chrome/content/rules/C9x.me.xml

```diff
@@ -0,0 +1,16 @@
+<!--
+	Nonfunctional hosts in *.c9x.me:
+
+	h: http redirect
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
```

Comment:
These are unnecessary, please delete comments that don't have corresponding subdomains.